src/shared/creds-util.c

```diff
@@ -83,6 +83,40 @@ int read_credential(const char *name, void **ret, size_t *ret_size) {
                         (char**) ret, ret_size);
 }
 
+int get_credential_user_password(const char *username, char **password, bool *is_hashed)
```

Comment:
in our codebase we generally name return parameters `ret_xyz`. i.e. please rename `password` → `ret_password`, and `is_hashed` → `ret_is_hashed`.

Reply:
See: https://systemd.io/CODING_STYLE/